scanner/hack/quay/go.mod

```diff
@@ -1,5 +1,3 @@
 module github.com/stackrox/rox/scanner/hack/quay
 
-go 1.22
```

Comment:
I'd usually say we should do:

```
go 1.23

toolchain 1.23.2
```

but I won't block this PR over this, especially when it's not the main module